src/libs/actions/Policy.js

```diff
@@ -638,9 +650,19 @@ function updateCustomUnitRate(policyID, currentCustomUnitRate, customUnitID, new
  * @param {String} policyID
  */
 function updateLastAccessedWorkspace(policyID) {
+    if (policyID === lastAccessedWorkspacePolicyID) {
```

Comment:
Not sure why we need this check? I thought we agreed to set this as `null`?

I am wondering if this is `lastAccessedWorkspacePolicyID` is already set  `policyID` even before this is called, we would end up with the same problem.